python/tank/authentication/shotgun_shared/saml2_sso.py

```diff
@@ -88,7 +125,7 @@ def __init__(self, window_title="SSO"):
         # will only limit the visit to the login flow, it is difficult to
         # display the warning only conditionnally. It is much simpler to hide
         # the message on our side.
-        css_style = base64.b64encode('div.browser_not_approved { display: none !important; }')
+        css_style = base64.b64encode("div.browser_not_approved { display: none !important; }")
```

Comment:
this raises a bit of flag for me - i don't fully understand the context here but keep in mind that if you are styling a web page, you should take into account that in the future, clients may run an old version of RV (emitting this css) while the web page or element that is being styled has moved on to new tech - there maybe a cost involved in retaining the backwards comp on the web side required to ensure that older client code is functional. Just a heads up.